packages/connect-widget/playwright.config.ts

```diff
@@ -0,0 +1,78 @@
+import * as dotenv from "dotenv";
+// somewhat what react-script does, tries to load .env.local, then .env
+process.env.CI
+  ? dotenv.config({ path: `.env.staging` }) // TODO still need to validate this approach when we enable tests on CI
+  : dotenv.config({ path: `.env.${process.env.REACT_APP_ENV_TYPE}.local` }).parsed ??
+    dotenv.config({ path: `.env.${process.env.REACT_APP_ENV_TYPE}` }).parsed ??
+    dotenv.config({ path: `.env.local` }).parsed ??
+    dotenv.config({ path: `.env` });
+// Keep dotenv import and config before everything else
+import { defineConfig, devices } from "@playwright/test";
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+  testDir: "./src/__tests__",
+  testMatch: /.*\.spec\.ts/,
```

Comment:
I just noticed we're using three diff extensions for tests:
- `.test.ts` - unit tests
- `.test.e2e.ts` - e2e tests on API
- `.spec.ts` - UI/Playwrights tests (that are being limited to E2E)

I know this was introduced before, but since we're adding this to a new project, it seems a good opportunity to review this.

Some issues I see with the current approach:
1. More than one extension for the same thing (`.spec.ts` and `.test.e2e.ts` refer to E2E tests);
2. Playwright tests can only be used for e2e (in theory, we could have unit tests for UI components as well).

One way to solve those would be to have Playwright accept `.test.ts` for unit tests (which we could run with `npx playwright test` through `npm run test`) and update the `npm run test:e2e` to run `npm run test -- "test\.e2e\.ts"` (so it only runs files matching that regex).

Thoughts?